### PR TITLE
Highlight the shown features if corresponding layer is visible

### LINKED ISSUE #1479: showFeatures should highlight
http://apps.geoportail.lu/exemple_api/apiv3_demo.html#example11

the feature is recentered on but not highlighted. 

I would propose the following: 
If the layer is present, it should be highlighted, otherwise not. (in this example parcels is present so it should be highlighted. if parcels not present, only zoom recenter)

### PR DESCRIPTION
**Pull request by pgiraud**
Fixes #1479

### CODE REVIEW

File: jsapi/src/main.js [hunk 1]
```diff
@@ -721,6 +766,8 @@ lux.Map.prototype.addFeature = function(json) {
     return;
   }
   var size = this.getSize();
+  // remove any highlighted feature
+  this.showLayer_.getSource().clear();
```
Comment (tonio) [inline jsapi/src/main.js]: `addFeature` may be called several times (cf [foreach](https://github.com/Geoportail-Luxembourg/geoportailv3/pull/1481/files#diff-80f6c764111465dfaa94038b75e98be0L695)),  `clear` should maybe better done in `showFeatures`…

Comment (pgiraud) [inline jsapi/src/main.js]: Fixed.

Comment (tonio) [inline jsapi/src/main.js]: `git push` ? (-:

Comment (pgiraud) [inline jsapi/src/main.js]: Pushed.

**Pull request merged**